### PR TITLE
[NCP-3180] added text for dns-01 challenge

### PR DESCRIPTION
**Pull request by jungho73**

### CODE REVIEW

File: en_US.json [hunk 1]
```diff
@@ -244,6 +244,22 @@
 	"certificate.deploy.inprogress.modal.title": "Deploying Certificate",
 	"certificate.deploy.success.modal.message": "The certificate has been successfully deployed.",
 	"certificate.deploy.success.modal.title": "Certificate Deployed",
+	"certificate.dns01chanllenge.error.another.instance": "Our system is busy validating another request. Please try again in a few minutes.",
+	"certificate.dns01chanllenge.error.exist.certificate": "You have an existing certificate that has exactly the same domains or certificate name you requested and isn't close to expiry.",
```
Comment (burtonquantil) [inline en_US.json:248]: I suggest changing "expiry" to "expiring"

File: en_US.json [hunk 2]
```diff
@@ -244,6 +244,22 @@
...
+	"certificate.dns01chanllenge.error.exist.timeout": "Time out to wait. Please try again",
```
Comment (burtonquantil) [inline en_US.json:249]: "Time out to wait" sounds really strange. What do you mean here?
"Timeout" is a word, but "Timeout to wait" doesn't make sense.

File: en_US.json [hunk 3]
```diff
@@ -244,6 +244,22 @@
...
+	"certificate.dns01chanllenge.fail.above.message": "We are unable to validate the following DNS records:",
+	"certificate.dns01chanllenge.fail.below.message": "Ensure that you have created the DNS records. If you have, you may need to wait longer for the DNS changes to propagate.",
+	"certificate.dns01chanllenge.fail.info.modal.title": "It failed to authenticate some domains",
```
Comment (burtonquantil) [inline en_US.json:252]: What is meant by "authenticate" in this context?
Maybe "DNS records not found for some domains"  would be better?

**Review by burtonquantil — CHANGES_REQUESTED** [review]
I posted some questions.

File: en_US.json (hunk 2 above)
Comment (jungho73) [reply en_US.json:249]: When issuing a certificate, waits for 30 minutes until clicking "Generate". But After that, if the user clicks "Generate", this message is displayed.

Comment (burtonquantil) [reply en_US.json:249]: I'm not sure I understand the first part. Are you saying the user has to wait 30 minutes until they can click the Generate button? That's a very long time.

Comment (jungho73) [reply en_US.json:249]: As we know, only one certbot can be run. Therefore, if there is no input after the message below is confirmed, it is set to terminate for the next user.
It says to wait up to 30 minutes, and the user can check and end at any time before that.

Please deploy a DNS TXT record under the name:
_acme-challenge.test.com.
with the following value:
VhtgnPIFy9ib0is2paJ8BpGSCgdfais-fUmaiD-7Y0c
You have 30 mins to finish this step...

Comment (burtonquantil) [reply en_US.json:249]: Are you saying the person ran out of the 30 minutes?
I suggest we be explicit and tell the user they have 30 minutes in advance.
For example:
"The setup of the DNS TXT records must be completed within 30 minutes. Please try again."

You can, of course, change it to an appropriate value if 30 is not correct.

"dns01chanllenge" should be "dns01challenge".

File: ar_SA.json [hunk 4]
```diff
@@ -244,6 +244,22 @@
...
+	"certificate.dns01chanllenge.modal.title.OK": "अस्तु",
+	"certificate.dns01chanllenge.modal.title.cancel": "निरसयतु",
+	"certificate.dns01chanllenge.modal.title.generate": "उद्- पद्",
+	"certificate.dns01chanllenge.modal.title.hostname": "होस्टनाम",
+	"certificate.dns01chanllenge.modal.title.txt.value": "TXT अभिलेखस्य मूल्यम्",
+	"certificate.dns01chanllenge.radio.title": "DNS-01 चुनौती",
+	"certificate.dns01chanllenge.requesting": "अनुरोधं कुर्वन्",
+	"certificate.dns01chanllenge.wait.above.message": "प्रमाणपत्रस्य होस्ट्नामानां स्वामित्वस्वामित्वं सत्यापयितुं कृपया निम्नलिखित DNS TXT अभिलेखान् योजयन्तु:",
+	"certificate.dns01chanllenge.wait.below.message": "एकदा भवता कृतं चेत्, निरन्तरं कर्तुं Generate नुदन्तु । ध्यानं कुर्वन्तु यत् नूतनानां DNS अभिलेखानां प्रसारणाय कतिपयानि निमेषाणि यावत् समयः भवितुं शक्नोति अतः वयं तान् पुष्टयितुं शक्नुमः ।",
+	"certificate.dns01chanllenge.wait.info.modal.title": "किं भवान् नूतनं प्रमाणपत्रं निर्मातुम् इच्छति ?",
```
Comment (chiraags) [inline ar_SA.json:262]: Jungho, the above language is Sanksrit and not Arabic.

ar_SA contains Arabic text. Can you please check if the BabelEdit is generating appropriate text for ar_SA?

This is not a blocker. But good to get it fixed.

Comment (jungho73) [reply ar_SA.json:262]: @chiraags 
BabelEdit selected "sa-In (Sanskrit - India)" for ar_SA.json automatically. Which one should we select?
And I updated BabelEdit to the new version recently. Is this probably the reason why it occurred?

Comment (chiraags) [reply ar_SA.json:262]: I haven't updated to the new version. Do you see an option to assign Arabic language?
SA stands for Saudi Arabia

Comment (jungho73) [reply ar_SA.json]: I fixed some text and changed the language for ar_SA.
Thanks

File: en_US.json (hunk 3 above)
Comment (jungho73) [reply en_US.json]: I fixed some text and changed the language for ar_SA.
Thanks

File: en_US.json (hunk 2 above)
Comment (jungho73) [reply en_US.json]: I fixed some text and changed the language for ar_SA.
Thanks

**Review by burtonquantil — APPROVED** [review]
OK to merge. Thanks for working on this task.

**Pull request merged**